### PR TITLE
Add plugin: Bridge admin panel

### PR DESCRIPTION
**Pull request by DeepSpaceHarbor**
#### Summary
I made a admin panel for hexo called [Bridge](https://github.com/DeepSpaceHarbor/hexo-bridge).
Would like to add the plugin to the official hexo website.

## Check List

**Please read and check followings before submitting a PR.**

- [x] I want to publish my plugin on Hexo official website.
    - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.

### CODE REVIEW

**Comment by DeepSpaceHarbor** [comment]
ping @SukkaW @YoshinoriN @stevenjoezhang, is everything ok with this pr? 
Do I need to make more changes before it gets approved & merged?

**Comment by jiangtj** [comment]
image

Can you remove this blank line?

**Comment by DeepSpaceHarbor** [comment]
@jiangtj done, I removed the empty line and resolved all of the conflicts.

File: source/_data/plugins.yml
Comment (jiangtj) [inline source/_data/plugins.yml:2833]: The blank line at the end of this file needs to be kept 😂

Comment (DeepSpaceHarbor) [reply source/_data/plugins.yml]: 4 attempts later, I did it. 😂 

Sorry about this,I thought yaml parsers are smart enough to find the end of file, even without the new line at the end.

File: source/_data/plugins.yml
Comment (jiangtj) [inline source/_data/plugins.yml:2819]: image


Plugin names seem to be package names, and I don't know if there's a convention for that? @hexojs/core

Comment (DeepSpaceHarbor) [reply source/_data/plugins.yml:2819]: Package name must start with hexo-* otherwise it won't be recognized. Given this, I thought it's ok use the real plugin name, without the hexo-* part. 
The documentation doesn't say anything about plugin names, if there's a rule that I wasn't aware of, let me know and I'll change the name.

Comment (DeepSpaceHarbor) [reply source/_data/plugins.yml]: In a effort to unblock this PR, I renamed the plugin to match the package name.

Comment (DeepSpaceHarbor) [reply source/_data/plugins.yml]: image

**Review by jiangtj — APPROVED** [review]
👍

**Pull request merged**
